skimage/transform/_geometric.py

```diff
@@ -763,39 +779,60 @@ class AffineTransform(ProjectiveTransform):
 
     Parameters
     ----------
-    matrix : (3, 3) array, optional
+    matrix : (D+1, D+1) array, optional
         Homogeneous transformation matrix.
     scale : (sx, sy) as array, list or tuple, optional
```

Comment:
Users may think that skimage does not *allow* scaling for >2D data, so let's point them to the other classes here that can achieve that.  Same for translation.

Reply:
@stefanv I've added a note to the main docstring, as I feel like it needed more context than the parameter list could provide. Let me know if you find it resolved.